src/store/version.ts

```diff
@@ -45,8 +45,8 @@ function clearAndSave() {
  * TODO: Configure this to be grabbed from package.json
  */
 export const currentVersion: Version = {
-    major: 1,
-    minor: 1,
+    major: 2,
+    minor: 3,
```

Comment:
Is this actually up to date? I seem to recall I set this for my version. Not that it matters too much as I don't think any of that is in Master.

Reply:
Yes, I updated it in a commit.